### PR TITLE
fix egl config chooser bug

### PR DESCRIPTION
**Pull request by dabingnn**
This PR is used to fix issue #8160

### CODE REVIEW

File: cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxActivity.java [hunk 1]
```diff
@@ -214,9 +221,79 @@ public void init() {
...
+								EGL10.EGL_NONE
+											};
+						//choose one can use
+						if(this.attribs[3] == 0)
+							egl.eglChooseConfig(display, defaultEGLattribs, configs, numConfigs[0], choosedConfigNum);
+						else
+							egl.eglChooseConfig(display, defaultEGLattribsAlpha, configs, numConfigs[0], choosedConfigNum);
+						if(choosedConfigNum[0] > 0)
+						{
+							Log.w(DEVICE_POLICY_SERVICE, "The EGLConfig can not be used for rendering, use a default one");
```
Comment (minggo) [inline cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxActivity.java:280]: It is better to write the config info, such as RGB565 or RGBA4444.

File: cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxActivity.java [hunk 2]
```diff
@@ -214,9 +221,79 @@ public void init() {
 	
     public Cocos2dxGLSurfaceView onCreateView() {
     	Cocos2dxGLSurfaceView glSurfaceView = new Cocos2dxGLSurfaceView(this);
-
-    	glSurfaceView.setEGLConfigChooser(this.glContextAttrs[0], this.glContextAttrs[1],this.glContextAttrs[2],
-    		this.glContextAttrs[3],this.glContextAttrs[4],this.glContextAttrs[5]);
+    	//this line is need on some device if we specify an alpha bits
+    	if(this.glContextAttrs[3] > 0) glSurfaceView.getHolder().setFormat(PixelFormat.TRANSLUCENT);
```
Comment (dumganhar) [inline cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxActivity.java:225]: What kinds of problem if don't assign `PixelFormat.TRANSLUCENT` to GLSurfaceView?

Comment (drelaptop) [reply cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxActivity.java:225]: not a reasonable adaptation, `need on some device` but case transparent issues on some other device.

refre to the discuss in forum: https://forum.cocos.org/t/cocos/81491

Comment (minggo) [reply cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxActivity.java:225]: Yep, i don't understand why need this line too. I think it is time to remove it.

**Pull request merged**